### PR TITLE
drt: cleanup workload runner scripts

### PR DESCRIPTION
**Pull request by ajstorm**
Cleanup the workload runner scripts to:

- Add the chaos test to systemd-run
- Fix a few errors in cct_tpcc_drop
- Add kv and tpcc scripts

Release note: none
Epic: none

### CODE REVIEW

File: pkg/cmd/drt/scripts/start_all_workloads.sh [hunk 1]
```diff
@@ -2,3 +2,4 @@ sudo systemd-run --working-directory=/home/ubuntu --service-type exec --collect
 sudo systemd-run --working-directory=/home/ubuntu --service-type exec --collect --unit cct_tpcc ./tpcc_run.sh
```
Comment (itsbilal) [inline pkg/cmd/drt/scripts/start_all_workloads.sh:2]: Let's add uids/gids for all of these, as they all need it now

Comment (ajstorm) [reply pkg/cmd/drt/scripts/start_all_workloads.sh]: Done.  I also added the schemachange workload.  PTAL.

**Review by itsbilal — APPROVED** [review]
LGTM

**Comment by ajstorm** [comment]
bors r=itsbilal

**Pull request merged**